index.html

```diff
@@ -379,11 +556,34 @@ <h5>SUBSCRIBE</h5>
                         <li><a href="">Digital Subscriptions</a></li>
                         <li><a href="">Games</a></li>
                         <li><a href="">Cookings</a></li>
+                        <li class="list_sep"><a href="">Election 2020</a></li>
+                     
+                        <li><a href="">Cookings</a></li>
+                        <li><a href="">Election 2020</a></li>
+                        <li><a href="">Cookings</a></li>
+                        <li><a href="">Election 2020</a></li>
+                        <li><a href="">Cookings</a></li>
                         <li><a href="">Election 2020</a></li>
                     </ul>
                 </div>
 
             </div>
+            <hr class="footer_endline">
+            <div class="footer-options">
+                <ul>
+                    <li><a href="">The New York Times Company</a></li>
+                    <li><a href="">NYTCo</a></li>
+                    <li><a href="">Contact Us</a></li>
+                    <li><a href="">Work with us</a></li>
+                    <li><a href="">Election 2020</a></li>
+                    <li><a href="">Cookings</a></li>
+                    <li><a href="">Election 2020</a></li>
+                    <li><a href="">Cookings</a></li>
+                    <li><a href="">Election 2020</a></li>
+                    <li><a href="">Cookings</a></li>
+                    <li><a href="">Subscribtion</a></li>
```

Comment:
There are a tags with empty href. Remember to use a hash(#) symbol in between the quotes, if there is no specific file path to include.

Reply:
Thank you for the pointer, we are surely working on the issues